### PR TITLE
Remove TLS and auth, code clean

### LINKED ISSUE #6: remove TLS and auth
TLS and auth should be removed from mailexporter and TLS and auth should be done by a tool specifically built for that (like authguard or an {nginx,apache,younameit}-reverseproxy) like in the rest of the prometheus-ecosystem.

### PR DESCRIPTION
**Pull request by dvrkps**
Closes #6

### CODE REVIEW

File: mailexporter.go
Comment (cherti) [inline mailexporter.go:298]: Good idea renaming this function, but I'd prefer `handleLateMail`, as this function also performs filesystem-operations and not just logging. Can you change that name?

**Review by cherti — CHANGES_REQUESTED** [review]
Thank you very much! Looks good to me except for one little detail. :)
After adressing that we can get this merged.

File: mailexporter.go
Comment (dvrkps) [reply mailexporter.go:298]: Sure.

**Review by cherti — APPROVED** [review]

**Comment by cherti** [comment]
Thank you very much, I'll merge once I had the time to test it.

**Pull request merged**
